woocommerce/woocommerce-android
# [Deposit Summary] Animate number change when the new comes from the backend

Closes: #10185


### Description

Adds change animation of the available and pending funds when a change comes from the backend

### Testing instructions

* Apply patch (or use your own method to adjust response from the backend)
[Patch_for_testing1.patch](https://github.com/woocommerce/woocommerce-android/files/13424860/Patch_for_testing1.patch)

* More -> Payments. Notice animation

The animation should not play on the other currencies, only the main one, and only once


### Images/gif


https://github.com/woocommerce/woocommerce-android/assets/4923871/2b3653e2-a850-4e98-9bef-5d2a3eae64b6
